packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -189,6 +212,52 @@ describe('TriggerManager', () => {
       wrapper.instance().handleSubmit()
       expect(wrapper.props().submitting).toEqual(true)
     })
+
+    it('should call saveAccount without account', () => {
+      const wrapper = shallowWithoutAccount()
```

Comment:
Nit : I find that it would preferable if this function would not two things at once (shallow rendering and passing the right props).

```
const wrapper = shallow(<TriggerManager {...props.noAccount} />)
```